Sources/Diagnostics/FileHandler.swift

```diff
@@ -127,6 +132,14 @@ actor FileHandler: FileHandlerType {
         try self.replaceHandler(with: tempURL)
     }
 
+    func fileSizeInKB() async throws -> Double {
+        let attributes = try FileManager.default.attributesOfItem(atPath: self.url.path)
```

Comment:
I took a look and I think there are no privacy labels that need to be added to use this but just fyi.